e2e/nomostest/new.go

```diff
@@ -68,6 +70,8 @@ func newOptStruct(testName, tmpDir string, ntOptions ...ntopts.Opt) *ntopts.New
 			// Default to 1m to keep tests fast.
 			// To override, use WithReconcileTimeout.
 			ReconcileTimeout: ptr.To(1 * time.Minute),
+			// Default to foreground to clean up test resources
+			DeletionPropagationPolicy: &foregroundDeletion,
```

Comment:
`ptr.To` should make the indirection easier

Reply:
done